server/game/game.js

```diff
@@ -1014,9 +1014,7 @@ class Game extends EventEmitter {
     }
 
     placeOnBottomOfDeck(card, options = { allowSave: true }) {
-        this.applyGameAction('placeOnBottomOfDeck', card, card => {
-            card.owner.moveCard(card, 'draw deck', { allowSave: options.allowSave, bottom: true });
-        });
+        card.owner.moveCardToBottomOfDeck(card, options.allowSave);
```

Comment:
Why this change? We should be moving away from all player / game methods like this and towards game actions.

Reply:
placeOnBottomOfDeck isn´t a "real" gameaction like returnCardToDeck, so i changed this helper method in game to reference the helper message in player, which in turn calls the real gameAction returnCardToDeck with the bottom: true option.
Using returnCardToDeck instead of placeOnBottomOfDeck enables Dupes to save against it (as the dupes trigger on onCardReturnedToDeck and not on placeOnBottomOfDeck). Furthermore, i changed PlankyTownTrader and the new card The Crone to reference cannotBeReturnedToDeck instead.
So all in all i try to remove placeOnBottomOfDeck as a gameAction and instead use returnCardToDeck gameaction.
While reviewing this, I noticed that i forgot to change ValarDohaeris´ reference to placeOnBottomOfDeck, will update that.

Reply:
I ended up removing `placeOnBottomOfDeck` entirely, rather than take this change. It was only really used in 3 cards. It should make Valar Dohaeris more simultaneous (but it will still have the problem with attachments until I finish my other branch). See: https://github.com/throneteki/throneteki/pull/3136/commits/9f605fcf9c116323adf1ca3923e782c14b314b34